README.md

```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
 
 ## Requirements
 
-At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04), it should be compatible with Mac OS and Windows as long as you have:
+At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04, Debian 12(bookworm)), it should be compatible with Mac OS and Windows as long as you have:
 
 * bash
 * git (to clone this repo)
 * make
 * python3 with pip3. Used for the `menuconfig`.
 * z88dk v2.2 (or later). Only its assembler, `z80asm`, is strictly required. The latest version of `z80asm` must be used as earlier versions don't have support for `MACRO`.
 
+Some commands are different between linux and windows, so we can't give you an example to build zos on windows, if you have any progress on it, please contact us.
```

Comment:
This isn't necessary as we stated we only tested on Windows at the moment. Some users reported that it assembles well using WSL, so in that case, it is not very different from Linux. 

Anyway, a Windows section in the future will be added

Reply:
Actually, I use WSL to build zos too, And I have try to build zos at another Subsystem. Windows here means a environment without a Subsystem like WSL, Msys, cygwin and so on. There is makefile support for windows, but The commands should be powershell-styled.

Reply:
We should add a Makefile.bat or add system prompt in Makefile.